Views/backend/connect/view/config/import/form.js

```diff
@@ -484,7 +484,7 @@ Ext.define('Shopware.apps.Connect.view.config.import.Form', {
                     fieldLabel: me.snippets.separateShippingLabel,
```

Comment:
don't you have to check this value in the basket?

Reply:
If this option is enabled, Connect plugin will show blue label "Separate dispatch" next to the product and separate line in basket, like vouchers.